nncf/common/factory.py

```diff
@@ -68,5 +75,25 @@ def create(model: TModel) -> Engine:
         if model_backend == BackendType.ONNX:
             from nncf.onnx.engine import ONNXEngine
             return ONNXEngine(model)
+        if model_backend == BackendType.OPENVINO:
+            from nncf.experimental.openvino_native.engine import OVNativeEngine
+            return OVNativeEngine(model)
         raise RuntimeError('Cannot create backend-specific engine'
                            'because {} is not supported!'.format(model_backend))
+
+
+# TODO(l-bat): Remove after fixing ticket: 100919
+def copy_model(model: TModel) -> TModel:
+    """
+    Function to create copy of the backend-specific model.
+
+    :param model: the backend-specific model instance
+    :return: Copy of the backend-specific model instance
+    """
+    model_backend = get_backend(model)
+    if model_backend == BackendType.OPENVINO:
+        return model.clone()
+    if model_backend == BackendType.ONNX:
+        return deepcopy(model)
+    raise RuntimeError('Cannot create backend-specific model copy '
```

Comment:
`deepcopy` should be used for other backends.

Reply:
done